static/app/components/inputGroup.tsx

```diff
@@ -44,7 +44,7 @@ export const InputGroupContext = createContext<InputContext>({inputProps: {}});
  *     <InputTrailingItems> … </InputTrailingItems>
  *   </InputGroup>
  */
-export function InputGroup({children}: React.HTMLAttributes<HTMLDivElement>) {
+export function InputGroup({children, className}: React.HTMLAttributes<HTMLDivElement>) {
```

Comment:
It looks like this was meant to work earlier, but was missed: https://github.com/getsentry/sentry/blob/master/static/app/components/textCopyInput.tsx#L93

No other callsites for `<InputGroup>` pass in a `className`, so i'm not worried about some styles getting ressurected someplace. One site comes close, ruleNode.tsx styles a `<NumberInput>` but that skips styling the group and styles its inner input directly.